### ISSUE TITLE
[v3.0.3.0b - Concept] - All files using the alert function in TWIG files

### ISSUE THREAD

**Issue by straightlight** [opencart/opencart#6540]
Could be harden to troubleshoot reported issues as well from the users. All files using the alert function from TWIG files:

admin/view/template/catalog/download_form.twig
admin/view/template/common/dashboard.twig
admin/view/template/common/developer.twig
admin/view/template/common/filemanager.twig
admin/view/template/common/security.twig
admin/view/template/customer/customer_approval.twig
admin/view/template/customer/customer_form.twig
admin/view/template/design/layout_form.twig
admin/view/template/design/theme.twig
admin/view/template/design/translation_form.twig
admin/view/template/localisation/geo_zone_form.twig
admin/view/template/marketing/affiliate_form.twig
admin/view/template/marketing/contact.twig
admin/view/template/marketplace/api.twig
admin/view/template/marketplace/cron.twig
admin/view/template/marketplace/event.twig
admin/view/template/marketplace/extension.twig
admin/view/template/marketplace/installer.twig
admin/view/template/marketplace/marketplace_info.twig
admin/view/template/marketplace/marketplace_list.twig
admin/view/template/sale/order_form.twig
admin/view/template/sale/order_info.twig
admin/view/template/sale/order_list.twig
admin/view/template/sale/voucher_form.twig
admin/view/template/sale/voucher_list.twig
admin/view/template/setting/setting.twig
admin/view/template/setting/store_form.twig
admin/view/template/tool/backup.twig
admin/view/template/user/api_form.twig
admin/view/template/extension/dashboard/chart_info.twig
admin/view/template/extension/dashboard/map_info.twig
admin/view/template/extension/feed/google_base.twig
admin/view/template/extension/fraud/ip.twig
admin/view/template/extension/payment/amazon_login_pay_order.twig
admin/view/template/extension/payment/bluepay_hosted_order.twig
admin/view/template/extension/payment/bluepay_redirect_order.twig
admin/view/template/extension/payment/eway_order.twig
admin/view/template/extension/payment/firstdata_order.twig
admin/view/template/extension/payment/firstdata_remote_order.twig
admin/view/template/extension/payment/g2apay_order.twig
admin/view/template/extension/payment/globalpay_order.twig
admin/view/template/extension/payment/globalpay_remote_order.twig
admin/view/template/extension/payment/klarna_checkout.twig
admin/view/template/extension/payment/klarna_checkout_order_ajax.twig
admin/view/template/extension/payment/laybuy_order.twig
admin/view/template/extension/payment/laybuy_transaction.twig
admin/view/template/extension/payment/pp_braintree.twig
admin/view/template/extension/payment/pp_payflow_iframe_order.twig
admin/view/template/extension/payment/pp_payflow_iframe_refund.twig
admin/view/template/extension/payment/pp_pro_iframe_order.twig
admin/view/template/extension/payment/realex_order.twig
admin/view/template/extension/payment/realex_remote_order.twig
admin/view/template/extension/payment/sagepay_direct_order.twig
admin/view/template/extension/payment/sagepay_server_order.twig
admin/view/template/extension/payment/securetrading_pp_order.twig
admin/view/template/extension/payment/securetrading_ws_order.twig
admin/view/template/extension/payment/squareup_recurring_buttons.twig
admin/view/template/extension/payment/worldpay_order.twig
catalog/view/theme/default/template/account/address_form.twig
catalog/view/theme/default/template/account/affiliate.twig
catalog/view/theme/default/template/account/edit.twig
catalog/view/theme/default/template/account/register.twig
catalog/view/theme/default/template/affiliate/register.twig
catalog/view/theme/default/template/checkout/checkout.twig
catalog/view/theme/default/template/checkout/guest.twig
catalog/view/theme/default/template/checkout/guest_shipping.twig
catalog/view/theme/default/template/checkout/payment_address.twig
catalog/view/theme/default/template/checkout/register.twig
catalog/view/theme/default/template/checkout/shipping_address.twig
catalog/view/theme/default/template/information/tracking.twig
catalog/view/theme/default/template/product/product.twig
catalog/view/theme/default/template/extension/payment/amazon_login_pay_address.twig
catalog/view/theme/default/template/extension/payment/authorizenet_aim.twig
catalog/view/theme/default/template/extension/payment/bank_transfer.twig
catalog/view/theme/default/template/extension/payment/bluepay_redirect.twig
catalog/view/theme/default/template/extension/payment/cardconnect.twig
catalog/view/theme/default/template/extension/payment/cheque.twig
catalog/view/theme/default/template/extension/payment/cod.twig
catalog/view/theme/default/template/extension/payment/eway_iframe.twig
catalog/view/theme/default/template/extension/payment/klarna_checkout_sidebar.twig
catalog/view/theme/default/template/extension/payment/perpetual_payments.twig
catalog/view/theme/default/template/extension/payment/pp_braintree.twig
catalog/view/theme/default/template/extension/payment/pp_payflow.twig
catalog/view/theme/default/template/extension/payment/pp_pro.twig
catalog/view/theme/default/template/extension/payment/sagepay_direct.twig
catalog/view/theme/default/template/extension/payment/sagepay_server.twig
catalog/view/theme/default/template/extension/payment/sagepay_us.twig
catalog/view/theme/default/template/extension/payment/web_payment_software.twig
catalog/view/theme/default/template/extension/payment/wechat_pay_qrcode.twig
catalog/view/theme/default/template/extension/payment/worldpay.twig
catalog/view/theme/default/template/extension/recurring/pp_express.twig
catalog/view/theme/default/template/extension/recurring/squareup.twig
catalog/view/theme/default/template/extension/total/coupon.twig
catalog/view/theme/default/template/extension/total/reward.twig
catalog/view/theme/default/template/extension/total/shipping.twig
catalog/view/theme/default/template/extension/total/voucher.twig

**Comment by danielkerr** [comment]
no idea what this is about
